NAMESPACE

```diff
@@ -1,5 +1,7 @@
 exportPattern("^cf.*")
 exportPattern("^plot.*")
+exportPattern("^plot_*")
```

Comment:
aren't we dealing with regular expression here? Shouldn't this be alread in `plot.*`?